### PR TITLE
feat: Convert to Flask server.

### LINKED ISSUE #148: Convert to Flask server

### PR DESCRIPTION
**Pull request by Magnesium12**
fixes #148

### CODE REVIEW

**Comment by Magnesium12** [comment]
I have also updated sentry_sdk to latest version in requirements

File: main.py [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
 from bot.models.github.event import GitHubEvent
 from bot.slack import SlackBot
 from bot.utils.log import Logger
 
+app = Flask(__name__)
 
-@get("/")
+
+@app.route("/")
 def test_get() -> str:
```
Comment (BURG3R5) [inline main.py:33]: Move this route to a new file `bot/views.py` using [this](https://stackoverflow.com/a/60441931)

File: main.py [hunk 2]
```diff
@@ -38,7 +38,7 @@ def test_get() -> str:
     return "This server is running!"
 
 
-@post("/")
+@app.route("/", methods=['POST'])
```
Comment (BURG3R5) [inline main.py:41]: Delete this route

File: main.py [hunk 3]
```diff
@@ -74,8 +75,12 @@ def manage_github_events():
     if event is not None:
         slack_bot.inform(event)
 
+    # After creation of Github-Webhook (through api),
+    # github sends a ping request to payload uri (for its verification) and expects an HTTP 200 OK response.
+    return make_response("Correct Payload URI", 200)
```
Comment (BURG3R5) [inline main.py:80]: Check if this is really necessary. I think if there's no `return` statement, it will still automatically return `200`.

Comment (BURG3R5) [reply main.py:80]: Remove this line and test the server. If it doesn't work, only then keep this here.

**Review by BURG3R5 — CHANGES_REQUESTED** [review]
Refer to comments for suggested changes

File: main.py (hunk 3 above)
Comment (Magnesium12) [reply main.py:80]: Without this return statement, flask is throwing (and responding with) the error: `TypeError: The view function for 'manage_github_events' did not return a valid response. The function either returned None or ended without a return statement.`
Also on returning something else like `True`, it throws: `TypeError: The view function did not return a valid response. The return type must be a string, dict, list, tuple with headers or status, Response instance, or WSGI callable, but it was a bool.`.
So, if we must return something then I think we should return this `Response` object.

**Review by BURG3R5 — DISMISSED** [review]

**Review by BURG3R5 — CHANGES_REQUESTED** [review]
Needs to be tested

**Comment by Magnesium12** [comment]
I have tested it and it can be merged.

**Pull request merged**